Integrations/integration-cisco-meraki.yml

```diff
@@ -1,10 +1,10 @@
 commonfields:
   id: Cisco Meraki
-  version: -1
+  version: 1
```

Comment:
not sure about this... I think all of them are -1 as they override current ones... please confirm